src/slice.c

```diff
@@ -761,6 +761,27 @@ static SEXP vec_chop_base(SEXP x, SEXP indices, struct vctrs_chop_info info);
 
 static SEXP vec_as_indices(SEXP indices, R_len_t n, SEXP names);
 
+// [[ register() ]]
+SEXP vctrs_chop_seq(SEXP x, SEXP starts, SEXP sizes, SEXP increasings) {
+  int* p_starts = INTEGER(starts);
+  int* p_sizes = INTEGER(sizes);
+  int* p_increasings = LOGICAL(increasings);
```

Comment:
Is it a bit strange to have increasings as plural? It sounds odd to my (admittedly non-native) ears.

Reply:
It is. If we had a way to make a compact seq on the R side I would just do `vec_chop(x, list(compact_seq()))` in the tests, but we don't so I figured this was the second best option. Since it is only used in testing I wasn't too worried about the name